libs/protocol-client/src/lib/graphql/queries.graphql

```diff
@@ -595,9 +595,17 @@ fragment ContributionFragment on Contribution {
   guilds {
     id
     guild_id
+    attestation_threshold
+    verification_status_id
+    verified
```

Comment:
This will be removed

Reply:
What do you mean?

Reply:
Same as above -- if this change is in #465 I'll resolve this after it's merged